app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -489,6 +536,55 @@ class StoryFragmentTest {
     }
   }
 
+  fun clickClickableSpan(textToClick: CharSequence): ViewAction {
```

Comment:
let make this function private as this is useable only in this file. If in future multiple test files need this function we will shift to utility.

Reply:
Also, could we add a reference link which can act like a guide to this function?
https://stackoverflow.com/questions/38314077/how-to-click-a-clickablespan-using-espresso

Reply:
Sure. I'll do it.

Reply:
Done.

Reply:
Resolving this as it is a small change.